src/config.rs

```diff
@@ -59,6 +64,13 @@ struct RawStyle {
     pub bold: bool,
 }
 
+
+#[derive(Copy, Clone, Default, Debug, Serialize, Deserialize, PartialEq, Eq)]
+struct RawDisplay {
+    #[serde(default = "default_pager")]
+    pub underline: bool,
```

Comment:
That should probably not be called `underline` :slightly_smiling_face: Also, is the struct used anywhere?